support/src/testFixtures/java/se/inera/intyg/common/support/facade/testsetup/model/config/ConfigRadioButtonMultipleCodeTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2022 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.common.support.facade.testsetup.model.config;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import se.inera.intyg.common.support.facade.model.config.CertificateDataConfigRadioMultipleCode;
+import se.inera.intyg.common.support.facade.model.config.CertificateDataConfigTypes;
+import se.inera.intyg.common.support.facade.model.config.Layout;
+import se.inera.intyg.common.support.facade.model.config.RadioMultipleCode;
+
+public abstract class ConfigRadioButtonMultipleCodeTest extends ConfigTest {
+
+    @Override
+    protected CertificateDataConfigTypes getType() {
+        return CertificateDataConfigTypes.UE_RADIO_MULTIPLE_CODE;
+    }
+
+    protected abstract List<RadioMultipleCode> getExpectedRadioMultipleCodes();
+    protected abstract Layout getExpectedLayout();
+
+    @ParameterizedTest
+    @MethodSource("getExpectedRadioMultipleCodes")
```

Comment:
Maybe we should not use an ParameterizedTest for this type. Since we dont use parameterized tests for other types outside of ToInternal.

Same logic could be used but with a small modification:

    @Test
    void shouldIncludeExpectedRadioMultipleCode() {
        final var config = (CertificateDataConfigRadioMultipleCode) getElement().getConfig();
        for (RadioMultipleCode radioMultipleCode : getExpectedRadioMultipleCodes()) {
            assertTrue(config.getList().contains(radioMultipleCode));
        }
    }